examples/catalog/lib/app_bar_bottom.dart

```diff
@@ -76,7 +76,7 @@ class _AppBarBottomSampleState extends State<AppBarBottomSample> with SingleTick
 }
 
 class Choice {
-  const Choice({ this.title, this.icon });
+  const Choice({this.title, this.icon});
```

Comment:
this change isn't great, but again i could live with it if it was the only change. obviously, having two changes that i could live with if they were the only change means that i don't really want to live with both. :-)

Reply:
It's easy to change. I didn't change it to minimize `dart_style` modifications (and simplify its maintenance).

If you forget all your other comments I modify that immediately. :)